_posts/2015-08-01-analysing-the-pomodoro-technique.md

```diff
@@ -0,0 +1,488 @@
+  reschedule the interruption, call back the person who interrupted you as agreed.
+* If It Lasts More Than 5-7 Pomodoros, Break It Down.
+  Complex activities should be divided into several activities.
+* If It Lasts Less Than One Pomodoro, Add It Up. Simple tasks can be combined.
+* Results Are Achieved Pomodoro after Pomodoro.
+* The Next Pomodoro Will Go Better.
+
+The benefits of this method are multiple:
+
+* Learning time, having a different experience of time.
+* Be more organized, prepare tasks and putting priorities
+* Be more aware, notice when you are getting distracted and strive to eliminate
+  distractions.
+* Be more analytic, learn from your way of working, find your weaknesses and set
+  new objectives to improve those.
+
+
+# Pomodoro apps: pros and cons
+
+Pomodoro is always listed in the top 2~3 best productivity methodologies. If
+you've been interested by productivity apps before, there's no doubt you've
+already heard of Pomodoro. Maybe you even tried an app, or several. And you
+might be part of the majority of people that didn't hook up to it and stopped
+using it very quickly.
+It might be a bit of your fault (lack of motivation, perseverance,
+self-discipline), but it's without doubt mostly due to the applications you
+used not being able to deliver a good user experience.
+
+When it comes to finding an application to use, it's extremely difficult to
+choose: There are more than a hundred pomodoro apps in Google's Play store.
+Maybe you found some recommendations and tried applications like 
+[Forest](https://play.google.com/store/apps/details?id=cc.forestapp&hl=en),
+[Pomodoro Challenge Timer](https://play.google.com/store/apps/details?id=com.wlxd.pomochallenge&hl=en),
+[PomoTodo](https://pomotodo.com/app/), [Pomodorium](http://www.pomodorium.com/),
+[Pomodoro.cc](http://pomodoro.cc/) or others. Yet you didn't manage to stick to
+it and build the habits to use Pomodoro regularly.
+
+One of the most important things to do before starting using Pomodoro is
+studying the method! The book is short (38 pages), althought it can take up to 5
+pomodoros to study it completely. If you do not understand what the method is
+about, what are the different steps to take, and levels to improve the
+technique, you won't improve with time, and eventually get disappointed.
+But for those who did their work and understand perfectly the method, I can bet
+that almost all of them got disappointed by existing apps, since none provide
+adequate tools to fully embrace that method, and following the method
+recommendations, they will have switched to paper and pencil to record their
+pomodoro and track their progress.
+
+Let's have a closer look of the applications metionned above:
+
+## Forest
+
+[ILLUSTRATION]
+
+Pros:
+
+* Excellent UI design
+* Gamified interface
+* Simple and Easy to use
+
+Cons:
+
+This is not a pomodoro application. It's a very simple timer, blocked on 30
```

Comment:
Hmm... the problem is maybe that people talk about that app as been a Pomodoro app, so when someone who learnt about the methodology is looking for an app, chances that he find Forest are quite high (and that will be a problem since it doesn't help for Pomodoro).